tachyon/math/polynomials/multivariate/multilinear_extension.h

```diff
@@ -166,7 +168,33 @@ H AbslHashValue(H h, const MultilinearExtension<Evaluations>& mle) {
   return H::combine(std::move(h), mle.evaluations());
 }
 
-}  // namespace tachyon::math
+}  // namespace math
+namespace base {
+
+template <typename Evaluations>
+class Copyable<math::MultilinearExtension<Evaluations>> {
+ public:
+  static bool WriteTo(const math::MultilinearExtension<Evaluations>& poly,
```

Comment:
```suggestion
  static bool WriteTo(const math::MultilinearExtension<Evaluations>& mle,
```
How about like this for consistency?